src/audio/audio.cpp

```diff
@@ -295,6 +295,13 @@ void Audio::playGroupAudio(int group, int peer, const int16_t* data,
         alSourcef(call.alSources[peer], AL_GAIN, outputVolume);
     }
 
+    size_t volume = 0;//data[1];
```

Comment:
Changing this to `qreal volume = 0.;` will make explicit cast in line 303 obsolete.